src/validations/basic/schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#/definitions/Validation",
+  "definitions": {
+    "Validation": {
+      "title": "Basic validation",
+      "type": "object",
+      "properties": {
+        "minScore": {
+          "title": "Minimum score",
+          "type": "number",
+          "minimum": 1
+        },
+        "useVotingStrategies": {
+          "description": "Use voting strategies",
+          "type": "boolean",
+          "default": false
+        },
+        "strategies": {
+          "title": "Strategies",
+          "type": "array",
+          "items": {
+            "title": "Strategy",
+            "type": "object",
+            "default": {
+              "name": "erc20-balance-of",
+              "network": "1",
+              "params": {}
+            },
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "network": {
+                "type": "string"
+              },
+              "params": {
+                "type": "object"
+              }
+            },
+            "required": ["name", "network", "params"]
```

Comment:
Why? There will be no way to not select a network on the UI

Reply:
But you cannot deselect it, so it will be selected and passed anyways ;)

Reply:
Please approve if there is nothing else

Reply:
> But you cannot deselect it, so it will be selected and passed anyways ;)

@ChaituVR after these changes I mean https://github.com/snapshot-labs/snapshot/pull/3775

Reply:
Sorry I was wrong about that, nevermind what I said you are right network should be optional